src/video/globlist.cxx

```diff
@@ -1,5 +1,5 @@
 #include "cvd/diskbuffer2.h"
-#include "cvd_src/config_internal.h"
```

Comment:
can remove this

Reply:
Below:

https://github.com/edrosten/libcvd/pull/69/files#diff-5352218138ea724cc13c024b825912db22115a676638477acbd62b025076ed7aL21

All platforms with glob now support BRACE and TILDE and the CMakeLists doesn't check anyway.